modular/blocksyncer/blocksyncer_options.go

```diff
@@ -374,6 +396,52 @@ func (b *BlockSyncerModular) prepareMasterFlagTable() error {
 	return nil
 }
 
+func (b *BlockSyncerModular) dbStatistics(ctx context.Context) {
+	duration := b.DataStatisticsDuration
+	if duration == 0 {
+		duration = 3600
```

Comment:
it looks a bit frequent.  "count" in sql will hold a exclusive lock which prevent newly records from inserting.

Reply:
ideally use read-only mysql instance